scripts/tools.yaml

```diff
@@ -50,5 +50,5 @@ Testssl:
   command: "git clone https://github.com/drwetter/testssl.sh.git"
   directory: tools/restricted/testssl
 GCPBucketBrute:
-  command: "git clone https://github.com/RhinoSecurityLabs/GCPBucketBrute.git"
-  directory: tools/restricted/gcpcbucketbrute
\ No newline at end of file
+  command: "git clone https://github.com/RhinoSecurityLabs/GCPBucketBrute.git ."
```

Comment:
remove the trailing `.`

Reply:
hey `.` is needed for cloning the contents of git repo in the current directory else it creates a folder naming GCPBucketBrute which will conflict with the current logic